docs/openapi/components/schemas/common/FSMAShipment.yml

```diff
@@ -0,0 +1,198 @@
+$linkedData:
+  term: FSMAShipment
+  '@id': https://w3id.org/traceability#FSMAShipment
+title: FSMA Shipment
+description: >-
+  "Information regarding an FSMA Shipment, a collection of KDEs (Key Data Elements) common to the
+  Shipping and Receiving CTEs (Critical Tracking Events). This schema is designed to meet reporting
+  requirements for the FDA Traceability Rule as specified here:
+  https://www.fda.gov/food/food-safety-modernization-act-fsma/fsma-proposed-rule-food-traceability"
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - FSMAShipment
+      - type: string
+        const: FSMAShipment
+  product:
+    title: Product
+    description: The product being tracked.
+    $ref: ./FSMAProduct.yml
+    $linkedData:
+      term: product
+      '@id': https://w3id.org/traceability#FSMAProduct
+  from:
+    title: From
+    description: The shipment's immediate origin. 
+    $ref: ./Place.yml
+    $linkedData:
+      term: from
+      '@id': https://schema.org/Place
+  to:
+    title: Ships To
+    description: The shipment's immediate destination.
+    $ref: ./Place.yml
+    $linkedData:
+      term: to
+      '@id': https://schema.org/Place
```

Comment:
id / type... what you mean to use here is:

The property with `id`: https://schema.org/location
Which supports the `type: https://schema.org/Place

This mistake is all over the place (not just in your PR here, but in many other schemas).

Here is an example of what I mean:

https://v.jsld.org/6DxQAm4Rf3J61R2L8HSynh9Hg741hKe4F4vJU1Z7VbdYnDPyx3WyzjuBcr5UMHpLtbis7Xsx7AjbWgYqaPK4fNh86qdiqvaJ7eN7GTTvRPiYoRR3aYziZ6z89KrYXDMCE1116SiKGXynhVJzvWBHcotgSemPyzaETLWCX5z2UeeMxsuRmgnn7s1qf8xRMaXDqT41EoDEZcps3L61c33oxjtqZZduTDhWD2Sw6xDMEYZATLmrh1rCE65Xwj6Ahpg8cpFmQXUVPygtAPShnZwbKXuzCZPfo9tYGkksEiNbrNLKB6YXaaKEdiDvxSXZaqodyF9nacxkgR58VB2Yn8PcRK2xvHZ2KH8JhmvPrjT8K9yYJ8riev2s9npwsGie2HgY2kxgpuKz6nFS2Ddad2QTxVBKGKR2eZjZXQGpc4wsY7qojJkCukn85Zj49Pw5kyPsLJiiqWnKpgRiW8

Reply:
The example is taken from: https://schema.org/Place#eg-0171

Reply:
```
_:c14n0 <https://schema.org/location> _:c14n3 ._:c14n0 <https://schema.org/location> _:c14n3 .
...
_:c14n3 <https://schema.org/address> _:c14n2 .
... 
_:c14n2 <https://schema.org/postalCode> "80209" .
```

Reply:
You only get "machine readability" if your JSON-LD /  RDF is correct... that means using `id` and `type` correctly.

Reply:
Yes @rhofvendahl this should be corrected

Reply:
@OR13 Thanks for this info! I definitely let laziness get the best of me with id/type in this PR, I'll make that a priority going forward.